docs/deployment.rst

```diff
@@ -27,9 +27,8 @@ We assume that you have root access to 3 Ubuntu 20.04 Server hosts / VMs. We rec
 Certificates (optional)
 .......................
 
-Certificates are needed for the Reducer and each of the Combiners to enable SSL for secure communication. 
-By default, FEDn will generate unsigned certificates for the reducer and each combiner using OpenSSL, but you can 
-also provide your own certificates. 
+You can provide certificates for each of the Combiners to enable secure gRPC connections. 
+FEDn can also generate unsigned certificates for using OpenSSL. The default deployment uses insecure channels, assuming that a production deployment will use a reverse proxy.  
```

Comment:
Client certificates (need for self-signed user provided certs) can be specified via the env variable FEDN_GRPC_ROOT_CERT_PATH

Reply:
I think the best thing would be if you uodated this section, since you implemented this.